test/functionalTest/cases/117_convop_using_standard_ops/getAllContextEntities.test

```diff
@@ -0,0 +1,608 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ExtraOp getAllContextEntities:  GET /v1/contextEntities
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. GET /v1/contextEntities and see it fail
+# 02. Update/APPEND E1/T1/A1
+# 03. GET /v1/contextEntities and see it work
+# 04. GET /v1/contextEntities?attributesFormat=object and see it work
+# 05. GET /v1/contextEntities?entity::type=T1 and see it work
+# 06. GET /v1/contextEntities?exist=entity::type and see it work
+# 07. GET /v1/contextEntities?entity::type=T2 and see it fail
+# 08. GET /v1/contextEntities?!exist=entity::type and see it fail
```

Comment:
I think it would be useful to include a variant of this in which `!exit=entity::type` works. It is a matter of include (e.g. as 9bis) an updateContext to include an entity without type, then check that `!exit=entity::type` works.